homework1/tests/task05.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from homework1.task05 import find_maximal_subarray_sum
+
+
+@pytest.mark.parametrize(
+    ["value", "value1", "expected_result"],
+    [
+        ([1, 3, -1, -3, 5, 3, 6, 7], 3, 16),
```

Comment:
([-1, -2, -3], 3, -1) - wrong answer for the test case

Reply:
fix.